lib/collection/src/collection/shard_transfer.rs

```diff
@@ -299,19 +299,13 @@ impl Collection {
         Ok(())
     }
 
-    /// Handles abort of the transfer
-    ///
-    /// 1. Unregister the transfer
-    /// 2. Stop transfer task
-    /// 3. Unwrap the proxy
-    /// 4. Remove temp shard, or mark it as dead
-    pub async fn abort_shard_transfer(
+    /// Return if it was a resharding transfer so it can be handled correctly (aborted or ignored)
+    pub async fn abort_shard_transfer_and_report_resharding(
```

Comment:
Is this only called `_and_report_resharding` because it returns a boolean defining whether it was a resharding transfer?